README.MD

```diff
@@ -10,24 +12,67 @@ The Magma Core Technical Charter is located in [CHARTER.md](CHARTER.md)
 
 ### Mailing List
 
+The TSC for Magma Core Foundation can be reached at their [mailing list](https://lists.magmacore.org/g/magma-tsc)
+
+
 ### Slack
 
+The Magma Core Foundation maintains a [Slack Workspace](https://join.slack.com/t/magmacore/shared_invite/zt-g76zkofr-g6~jYiS3KRzC9qhAISUC2A) for communication and collaboration. The Magma Core Foundation [Slack Workspace](https://join.slack.com/t/magmacore/shared_invite/zt-g76zkofr-g6~jYiS3KRzC9qhAISUC2A) is open for anyone to join.
+
+
 ### Calendars and Meetings
 
+The Magma Core Foundation maintains a [public calendar](https://calendar.google.com/calendar/embed?src=c_gbiu1t7a67ika1th2smldeh19s%40group.calendar.google.com&ctz=America%2FChicago) for TSC and community meetings. These meetings are open for anyone to join.
+
+
 ### CLA
 
 Everyone must sign the CLA prior to making a contribution. You may either sign the CLA on your own, or your company can sign it for you. At a high level, if you aren't covered by a CLA, you will be notified the first time you open a PR. Please see our [documentation on the process](./CLA.md).
 
 ## Members
 
+The voting members of the Magma Core Foundation TSC are listed [here](https://github.com/magma/Governance/wiki/Magma-TSC-Members)
```

Comment:
Here and throughout the changes: use descriptive hyperlink text, i.e. not "here" https://magma.github.io/magma/docs/next/contributing/contribute_conventions#all-documentation